hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+
+ - Tx power limits
+   The RF output power is defined as the mean equivalent isotropically radiated power(EIRP) of the equipment during a transmission burst. 
+   The maximum allowed output power is +20dBm.
+
+ - Occupied Channel Bandwidth
+   It is defined as bandwidth containing 99% energy of the output power.
+   For FHSS systems:
+   - For equipment with EIRP ≤ +10 dBm, OCBW is simply declared by the supplier (Section 4.3.1.7.2)
+   - For equipment with EIRP > +10 dBm, OCBW is declared by the supplier, but it should be less than 5 MHz (Section 4.3.1.7.2)
+   For wideband digital modulation systems:
+   - For equipment with EIRP > +10 dBm, OCBW declared by the supplier, but less than 20 MHz (Section 4.3.2.6.2)
+
+ - TX Power Spectral Density
+   - Applies only to wideband digital modulation systems, and not FHSS systems
+   - PSD(max) ≤ +10 dBm / MHz (Section 4.3.2.2.2)
+
+ - TX Unwanted Emissions in Out-of-Band Domain
+   - OOB Domain defined for “near-by” signals (relative to edges of frequency band)
+   - Within 2*BW of 2.400-2.4835 GHz band edges, where BW = Occupied Channel Bandwidth
+   - At greater frequency offsets, the Spurious Domain limits apply
+   - OOB Domain Limits:
+     - -10 dBm/MHz EIRP, within 1*BW offset from band edge (Section 4.3.1.8.2)
+     - -20 dBm/MHz EIRP, within 1*BW to 2*BW offset from band edge (Section 4.3.1.8.2)
+
+ - TX Unwanted Emissions in Spurious Domain
+   - Spurious Domain defined for “far-away” signals (relative to edges of frequency band)
+   - More than 2*BW away from band edges,Where “BW” is the Occupied Channel Bandwidth (but minimum value of 1 MHz)
+   - At lesser frequency offsets, the Out-of-Band Domain limits apply
+   - Spurious Domain Limits: (Section 4.3.1.9.2): 1-12.75 GHz, -30 dBm EIRP, RBW=1MHz
+
+ - Rx sensitivity level
+   No general requirements. But it is defined in Zigbee and BLE protocols respectively.
+
+ - Blocking
+   - Desired signal at -30 dBm, Interferer at -30 dBm (CW) (Section 4.3.1.11.2)
+     - Relative blocking level much easier than BTLE spec
+     - Absolute blocking level slightly harder than BTLE spec
+   - But not applicable for systems with TX EIRP < +10dBm. I.e., not applicable to BTLE
+
+ - RX Spurious Emissions
+   - 30-1000 MHz, -57 dBm ERP, BW=100kHz (Section 4.3.1.10.2)
+   - 1-12.75 GHz, -47 dBm EIRP, BW=1MHz (Section 4.3.1.10.2)
+
+# 5. Customer regulatory cerfitication flow
+Since there are SoCs, SiP modules, and PCB modules in Silicon Labs product catalog. The cerfification work may different among them. It only lists the basical certification flows in this chapter.
+## 5.1. Silicon Labs provides documents for certification
+Before the certification process, Silicon Labs can provides a very exclusive documents for customer adoption. The documents are:
```

Comment:
Change "provides a very" to "provide very".

Reply:
Corrected.